gpu_monitor.py

```diff
@@ -281,10 +336,11 @@ def main(argv):
 
         gpu_infos = get_gpu_infos(nvidiasmi)
 
-        if args.list:
-            print_gpu_infos(server, gpu_infos, run_ps, run_get_real_names,
-                            filter_by_user=args.user,
-                            translate_to_real_names=args.finger)
+        if args.list or args.taskset:
```

Comment:
Once taskset is factored into its own method, there should be `if args.taskset: set_task_affinities(...)` before `if args.list`.